kfdefs/overlays/moc/smaug/opf-jupyterhub/alerts.yaml

```diff
@@ -26,3 +26,53 @@ spec:
             < 0.1
           labels:
             severity: critical
+    - name: SLOs-probe_success
```

Comment:
@mubariskhan96 let's remove the alerts from this PR and create a new PR for those